_api_app/app/Http/routes.php

```diff
@@ -31,13 +31,13 @@
     $app->patch('site-template-settings', ['as' => 'site_template_settings', 'uses' => 'Sites\SiteTemplateSettings\SiteTemplateSettingsController@update']);
 
     $app->group(['prefix' => 'v1', 'namespace' => 'App\Sites\Sections'], function() use ($app) {
-        $app->post('sections', ['as' => 'sections', 'uses' => 'SiteSectionsController@create']);
```

Comment:
Let's keep the URLs like we keep namespaces, with slashes, so we can see what belongs to what. In this case, the sections URL should be `sites/sections`, because it's under the sections endpoint in hierarchy (the sections are in sites, so sections endpoint is in sites endpoint)